astroid/manager.py

```diff
@@ -357,6 +358,21 @@ def bootstrap(self):
         raw_building._astroid_bootstrapping()
 
     def clear_cache(self):
-        """Clear the underlying cache. Also bootstraps the builtins module."""
+        """Clear the underlying caches. Also bootstraps the builtins module."""
+        # pylint: disable=import-outside-toplevel
```

Comment:
Is it because of cyclic import ? We could state so 
```suggestion
        # pylint: disable=import-outside-toplevel
        # import here because of cyclic imports
```